src/main/java/mate/jdbc/lib/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String sqlQueryForAddManufacturerToDb
+                = "INSERT INTO manufacturers(name, country) VALUES(?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection
+                            .prepareStatement(sqlQueryForAddManufacturerToDb,
+                                    Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long id = resultSet.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacturer to DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String sqlQueryForGetDataFromDb
+                = "SELECT * FROM manufacturers WHERE is_deleted = false AND id=(?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection
+                         .prepareStatement(sqlQueryForGetDataFromDb)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                Manufacturer manufacturer = initializeManufacturer(resultSet);
+                return Optional.of(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't read info from DB by id=" + id);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String sqlQueryForGetDataFromDb
+                = "SELECT * FROM manufacturers WHERE is_deleted = false";
+        List<Manufacturer> allDataInDb = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement statement = connection.createStatement()) {
+            ResultSet resultSet = statement.executeQuery(sqlQueryForGetDataFromDb);
+            while (resultSet.next()) {
+                Manufacturer manufacturer = initializeManufacturer(resultSet);
+                allDataInDb.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't read info from DB");
+        }
+        return allDataInDb;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String sqlQueryForUpdateDataById
+                = "UPDATE manufacturers SET name = (?), country = (?) "
+                + "WHERE id = (?) is_deleted = false";
```

Comment:
```suggestion
        String query
                = "UPDATE manufacturers SET name = (?), country = (?) "
                + "WHERE id = (?) is_deleted = false";
```
let's rename this in all places